src/main/java/org/frankframework/frankdoc/model/FrankElementFilters.java

```diff
@@ -63,7 +62,6 @@ private FrankElementFilters() {
 		
 		// Exclude classes that should only be used in internal configurations
 		excludeFilters.add("nl.nn.adapterframework.doc.IbisDocPipe");
-		excludeFilters.add("nl.nn.adapterframework.webcontrol..*");
 		excludeFilters.add("nl.nn.adapterframework.pipes.CreateRestViewPipe");
```

Comment:
Ik heb gecontroleerd dat er geen webcontrol classes in de XSDs verschijnen door dit filter weg te laten.